### PR TITLE
Update io.hpp to replace get_x() with get_z()

### PR DESCRIPTION
**Pull request by nitink25**
### Description

1. Typo corrected : `get_x()` has been replaced with `get_z()`.  
2. Indentation corrected to adhere to less than 100 characters per line.

### CODE REVIEW

File: include/boost/astronomy/coordinate/io.hpp [hunk 1]
```diff
@@ -24,7 +24,7 @@ template
 >
 std::ostream& operator<< (std::ostream &out, cartesian_representation< CoordinateType, XQuantity, YQuantity, ZQuantity> const& point)
 {
-    out << "Cartesian Representation ( " << point.get_x() << " , " << point.get_y() << " , " << point.get_x() << " )";
+    out << "Cartesian Representation ( " << point.get_x() << " , " << point.get_y() << " , " << point.get_z() << " )";
```
Comment (lpranam) [inline include/boost/astronomy/coordinate/io.hpp]: Break this line to fit in within the limit of 100 characters

**Review by lpranam — CHANGES_REQUESTED** [review]

File: include/boost/astronomy/coordinate/io.hpp (hunk 1 above)
Comment (nitink25) [reply include/boost/astronomy/coordinate/io.hpp]: Sorry about repeatedly missing that rule...Will adhere to it in the future.

File: include/boost/astronomy/coordinate/io.hpp [hunk 2]
```diff
@@ -22,9 +22,11 @@ template
     class YQuantity,
     class ZQuantity
 >
-std::ostream& operator<< (std::ostream &out, cartesian_representation< CoordinateType, XQuantity, YQuantity, ZQuantity> const& point)
+std::ostream& operator<< (std::ostream &out, cartesian_representation
+	<CoordinateType, XQuantity, YQuantity, ZQuantity> const& point)
 {
-    out << "Cartesian Representation ( " << point.get_x() << " , " << point.get_y() << " , " << point.get_x() << " )";
+    out << "Cartesian Representation ( "
```
Comment (lpranam) [inline include/boost/astronomy/coordinate/io.hpp:28]: I think this style would make code more readable

```
out << "Cartesian Representation ( " 
    	<< point.get_x() << " , " 
        << point.get_y() << " , " 
        << point.get_z() << " )";
```

File: include/boost/astronomy/coordinate/io.hpp [hunk 3]
```diff
@@ -37,9 +39,11 @@ template
     class LonQuantity,
     class DistQuantity
 >
-std::ostream& operator<< (std::ostream &out, spherical_equatorial_representation< CoordinateType, LatQuantity, LonQuantity, DistQuantity> const& point)
+std::ostream& operator<< (std::ostream &out, spherical_equatorial_representation
+	<CoordinateType, LatQuantity, LonQuantity, DistQuantity> const& point)
 {
-    out << "Spherical Equatorial Representation ( " << point.get_lat() << " , " << point.get_lon() << " , " << point.get_dist() << " )";
+    out << "Spherical Equatorial Representation ( "
```
Comment (lpranam) [inline include/boost/astronomy/coordinate/io.hpp:47]: Change the style as mentioned above

File: include/boost/astronomy/coordinate/io.hpp [hunk 4]
```diff
@@ -52,9 +56,11 @@ template
     class LonQuantity,
     class DistQuantity
 >
-std::ostream& operator<< (std::ostream &out, spherical_representation< CoordinateType, LatQuantity, LonQuantity, DistQuantity> const& point)
+std::ostream& operator<< (std::ostream &out, spherical_representation
+	<CoordinateType, LatQuantity, LonQuantity, DistQuantity> const& point)
 {
-    out << "Spherical Representation ( " << point.get_lat() << " , " << point.get_lon() << " , " << point.get_dist() << " )";
+    out << "Spherical Representation ( "
```
Comment (lpranam) [inline include/boost/astronomy/coordinate/io.hpp:66]: Change the style as mentioned above

**Review by lpranam — CHANGES_REQUESTED** [review]
Probably the last required change

File: include/boost/astronomy/coordinate/io.hpp [hunk 5]
```diff
@@ -22,9 +22,13 @@ template
...
     class ZQuantity
 >
-std::ostream& operator<< (std::ostream &out, cartesian_representation< CoordinateType, XQuantity, YQuantity, ZQuantity> const& point)
+std::ostream& operator<< (std::ostream &out, cartesian_representation
+	<CoordinateType, XQuantity, YQuantity, ZQuantity> const& point)
 {
-    out << "Cartesian Representation ( " << point.get_x() << " , " << point.get_y() << " , " << point.get_x() << " )";
+    out << "Cartesian Representation ( " 
+    	    << point.get_x() << " , " 
+            << point.get_y() << " , "
```
Comment (nitink25) [inline include/boost/astronomy/coordinate/io.hpp]: Due to some reason, indentation between line `29` and `30` appears different in this preview. But that's not the case in the original file. So not a problem.

**Review by lpranam — APPROVED** [review]

**Pull request merged**